mteb/tasks/__init__.py

```diff
@@ -8,3 +8,4 @@
 from .Retrieval import *
 from .STS import *
 from .Summarization import *
+from .InstructionReranking import *
```

Comment:
I think the folder structure has also changed so there should probably be an English folder cc @KennethEnevoldsen ?

Also, why `InstructionReranking` and not `InstructionRetrieval` or `FollowIR`?

Reply:
This is a good question.  We could do InstructionRetrieval but in the case of FollowIR it's actually reranking not retrieval (1000 docs per query).  

There's another paper concurrent with mine that took an automated approach to data creation from [KAIST](https://github.com/kaistAI/InstructIR/tree/main).  I think I will see if they're interested in merging into MTEB also -- people always like to evaluate on more than one dataset anyways.  For that reason, I was thinking of keeping it more broad, so that it wasn't just FollowIR.

I think InstructionalRetrieval is more broad (which I like so that people don't think it can only be done for reranking) but InstructionReranking is technically more accurate. TBH I'm happy with either, both are good.  Do you have a preference @Muennighoff?